docs/framework/filesystem.md

```diff
@@ -130,13 +130,16 @@ echo Civi::resources()->addScriptFile('civicrm.bower', 'jquery/dist/jquery.min.j
 
 ## Local data files
 
-CiviCRM also needs a directory for storing volatile data files, such as 
-logs, caches, and uploads.  This directory is located outside
+CiviCRM also needs directories for storing volatile data files, such as 
+logs, caches, and uploads. These directories are located outside
 the main codebase -- in a location that can be safely preserved during
 upgrades.
 
-This folder is generically referred to as `[civicrm.files]`, but the actual
-path is chosen to align with each CMS's conventions.
+Public data folders use `[civicrm.files]` and public data folders use 
+`[civicrm.private]` are for private data folders. The actual path is chosen
+to align with each CMS's conventions and typically `[civicrm.private]` would
+be the same as `[civicrm.files]` while still allowing sysadmins to have the
+ability to manage paths precisely.
 
```

Comment:
This section needs rewording - I can't suggest changes here because it's on separate lines.

Something like:

> CiviCRM provides two main file storage helpers:
>
> * `[civicrm.files]` - Intended to store files which can safely live within the files directory of your CMS, within your webroot.
> * `[civicrm.private]` - Intended to store files which could be stored outside of your webroot for enhanced security.
>
> The actual path is chosen to align with the conventions of each CMS and typically `[civicrm.private]` will refer to the same directory as `[civicrm.files]` whilst providing the ability to manage paths in a more granular fashion should the need arise.